modules/tinymce/src/core/main/ts/api/dom/ControlSelection.ts

```diff
@@ -203,7 +203,7 @@ const ControlSelection = (selection: EditorSelection, editor: Editor): ControlSe
     width = width < 5 ? 5 : width;
     height = height < 5 ? 5 : height;
 
-    if ((isImage(selectedElm) || isMedia(selectedElm)) && Options.getResizeImgProportional(editor) !== false) {
+    if ((isImage(selectedElm) || isMedia(selectedElm) || isUcVideo(selectedElm)) && Options.getResizeImgProportional(editor) !== false) {
```

Comment:
I wonder if we should add some form of predicate registry API to ControlSelection so plugins can add/remove what should be resizables or not. But then there is a risk that customers would start resizing all kinds of things that we don't want to be resizable. 🤔

Reply:
I think we already have a Jira to refactor this and make this more generic way, but I don't remember the code right now :(